helm/ako/crds/ako.vmware.com_aviinfrasettings.yaml

```diff
@@ -105,8 +105,6 @@ spec:
                 - shardSize
               nsxSettings:
                 properties:
-                  project:
```

Comment:
@abhishekbsingh: Removal of existing field will break existing aviinfra crd objects (it these fields being used) in k8 cluster. So for this we will require version upgrade and crd conversion strategy to move existing objects to new version.